src/cli.rs

```diff
@@ -61,9 +64,13 @@ pub struct Rover {
     #[serde(serialize_with = "option_from_display")]
     log_level: Option<Level>,
 
-    /// Specify Rover's output type
-    #[arg(long = "output", default_value = "plain", global = true)]
-    output_type: OutputType,
+    /// Specify Rover's format type
```

Comment:
```suggestion
    /// Specify the format of the data printed to stdout by Rover's commands
```

What do you think about this updated help text?

cc @StephenBarlow as well for wording